svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/+layout.svelte

```diff
@@ -6,17 +6,10 @@
   const { repoInfo } = data;
 </script>
 
+<RepoHeader repo={repoInfo} />
 <div class="container">
-  <RepoHeader repo={repoInfo} />
-  <div class="main">
-    <slot />
-  </div>
+  <slot />
 </div>
 
 <style lang="scss">
-  .container {
-    .main {
-      padding: 1.5rem;
```

Comment:
Did we need this `padding: 1.5rem` though?

Reply:
No it was not needed

Reply:
No it wasnt needed